apis/dataprotection/v1alpha1/types.go

```diff
@@ -261,3 +262,28 @@ type EncryptionConfig struct {
 	// +kubebuilder:validation:Required
 	PassPhraseSecretKeyRef *corev1.SecretKeySelector `json:"passPhraseSecretKeyRef"`
 }
+
+type SelectiveParametersSchema struct {
```

Comment:
This structure I tend to just named ParametersSchema, or to be more specific, since it's used for actionSet, it could be named ActionSetParametersSchema.

Reply:
agree

Reply:
done